apps/src/applab/Reducers.js

```diff
@@ -0,0 +1,42 @@
+/** @file Redux reducer functions for App Lab.
+ *  @see http://redux.js.org/docs/basics/Reducers.html */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var ActionType = require('./Actions').ActionType;
+
+function rootReducer(state, action) {
+  if (typeof state === 'undefined') {
+    state = {
+      assetUrl: function () {}
+    };
+  }
```

Comment:
A common form I see for reducer's is something like this (ES6):
```
const initialState = {
  assetUrl: function () {}
};

function rootReducer(state = initialState, action) {
 ...
}
```
We could accomplish similar in ES5 by doing this:
```
var initialState = {
  // we could also initialize all of our other props to null/undefined here so that we have
  // an easy way to see the shape of  our state
  assetUrl: function () {}
};

function rootReducer(state, action) {
  state = state || initialState; // we don't have to worry about valid falsey states
 ...
}